### PR TITLE
updated ABOUT.md to include more "why"

### PR DESCRIPTION
**Pull request by dsudia**
Added more information about the language, its tooling, and why someone would want to learn it. Happy to hear feedback on it!

### CODE REVIEW

File: docs/ABOUT.md [hunk 1]
```diff
@@ -1 +1,5 @@
 "Swift is an alternative for the Objective-C language that employs contemporary programming language theory concepts and strives to present a simpler syntax. Swift introduces true named parameters and retains key Objective-C concepts, including protocols, closures and categories, often replacing former syntax with cleaner versions and allowing these concepts to be applied to other language structures, like enums." -  https://en.wikipedia.org/wiki/Swift_(programming_language)
+
+Swift is a great first language to learn because it was designed to be easily readable and writable. Swift was created by Apple as an evolution of Objective-C with those goals in mind. Apple also provides lots of high-quality documentation and tools for Swift, which make it a good place to start. However, one thing to know is that the only tool available right now to code in Swift (XCode) is only available for Macs, so if you don't own one, you won't be able to work with Swift.
```
Comment (masters3d) [inline docs/ABOUT.md]: Swift was open sourced last year and is avalible on linux https://swift.org/download/

**Comment by masters3d** [comment]
Hi @dsudia! Thanks for the PR. 
I'm afraid that your new additions are not neutral and do not add much to the "about" section.
Swift is changing a lot with version 3 https://github.com/apple/swift-evolution
I'd like to make this track independent of Xcode once swift 3 is avalible. 
Once that happens I think I will be a good time to update the about section. 

I do think that adding these links to the About section would be beneficial:
https://swift.org/about/
https://github.com/apple/swift-evolution

**Comment by masters3d** [comment]
In addition, Microsoft just released the ubuntu subsystem that should allow Swift to run on windows. 
https://insights.ubuntu.com/2016/04/14/howto-ubuntu-on-windows-2/

Im closing this for now, we should revisit by the end of the year.

**Comment by dsudia** [comment]
Thanks, that's some news I hadn't heard. I'll check back later to see how it's developed and see if I can contribute anything then.

**Pull request closed without merging**